onnxruntime/core/platform/env.h

```diff
@@ -232,6 +251,10 @@ class Env {
   // Returns empty string if there is no such environment variable available
   virtual std::string GetEnvironmentVar(const std::string& var_name) const = 0;
 
+  // \brief provide an interface to set system info to simulate numa architectures
+  virtual std::vector<int32_t> GetSystemInfo() = 0;
```

Comment:
It's not clear what these return. Besides, are they really needed in the public interface since they're needed for the test only? You can potentially derive a new class from WindowsEnv, place it in threadpool_test.cc and use that. See https://github.com/microsoft/onnxruntime/blob/48b2ec944c89c2d3ec22ee5cfc2285851e02bb04/onnxruntime/test/framework/inference_session_test.cc#L2436 as an example.